### PR TITLE
google auth

### PR DESCRIPTION
**Pull request by KABANOFESTO**

### CODE REVIEW

File: .example.env [hunk 1]
```diff
@@ -0,0 +1,14 @@
+DBLINK = postgres://users_5p3h_user:gNzkEQhb4BiBs4Ivd19gPkBDjTLFF4kc@dpg-cf9ngp4gqg47p3vrnh00-a.oregon-postgres.render.com/users_5p3h
```
Comment (Rukundojoseph) [inline .example.env]: don't share this file.
with the .env information.

Comment (KABANOFESTO) [reply .example.env]: ooh yeah you are sure and  I already fix it thank you

**Review by TresorRw — COMMENTED** [review]
Don't put environment variables to online codebase.

**Review by Rukundojoseph — DISMISSED** [review]

**Review by TresorRw — DISMISSED** [review]

File: src/controllers/googleAuthController.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import User from '../models/User'
+import bcrypyt from 'bcrypt'
+import jwt from "jsonwebtoken"
+const googleAuth = async (req, res) => {
+    // console.log(req.user)
```
Comment (EmyRukundo) [inline src/controllers/googleAuthController.ts]: kindly remove this

**Review by EmyRukundo — DISMISSED** [review]

**Comment by key-joshua** [comment]
Your pull request still has un-resolved conflicts.

**Comment by key-joshua** [comment]
Your pull request still has unresolved conflicts.

**Review by fabrice-rw — DISMISSED** [review]
There're some conflicts to be resolved

**Review by EmyRukundo — CHANGES_REQUESTED** [review]
solve the conflict plz and I do not see the test codes

**Pull request closed without merging**